CHANGELOG.md

```diff
@@ -21,6 +27,13 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 - Helpers: Rename `labels.common` to `ingress-nginx.labels`. ([#358](https://github.com/giantswarm/nginx-ingress-controller-app/pull/358))
 - Templates: Align hook annotations, namespaces & indention. ([#359](https://github.com/giantswarm/nginx-ingress-controller-app/pull/359), [#361](https://github.com/giantswarm/nginx-ingress-controller-app/pull/361))
 - Templates: Align `ValidatingWebhookConfiguration`. ([#363](https://github.com/giantswarm/nginx-ingress-controller-app/pull/363))
+- Webhook: Disable privilege escalation. ([#366](https://github.com/giantswarm/nginx-ingress-controller-app/pull/366))
+- Webhook: Align image concatenation. ([#366](https://github.com/giantswarm/nginx-ingress-controller-app/pull/366))
+- Webhook: Align values.yaml. ([#366](https://github.com/giantswarm/nginx-ingress-controller-app/pull/366))
+
+### Removed
+
+- Webhook: Remove `controller.admissionWebhooks.patch.backoffLimit`. ([#366](https://github.com/giantswarm/nginx-ingress-controller-app/pull/366))
```

Comment:
Can we add a reason why this isn't necessary anymore?

Reply:
Yeah, sure. I'll do. Actually we always used the default value of 6 and I wanted to try if it's working without explicitly setting it so there one diff less we need to take care of when aligning with upstream.

Reply:
Done!